app/map_styles/polygon.xml

```diff
@@ -517,6 +517,10 @@
             <Filter>[current_landuse_order] = "Minerals"</Filter>
             <PolygonSymbolizer fill="#53f5dd" />
         </Rule>
+        <Rule>
+            <Filter>[current_landuse_order] = "Unknown buildings"</Filter>
```

Comment:
The rule here has "Unknown buildings" and the migrations add a type for "Unclassified buildings"

Reply:
@mz8i Fixed, rename in https://github.com/colouring-cities/colouring-london/pull/711/commits/f97fba7f3d6ee3b009d1ee176dacc3172226a16a was unfinished